src/requests/flow.ts

```diff
@@ -167,6 +173,7 @@ export async function createFlow(
       flowName: args.name,
       data: args.data,
       status: args.status || "offline",
+      description: "",
```

Comment:
I think it might make more sense to consider this a nullable field - current flows don't have descriptions and I don't think a description will be mandatory for a flow.